### PR TITLE
feat: add `--window-title` support

### PR DESCRIPTION
**Pull request by andresatierf**

### CODE REVIEW

File: lua/silicon.lua [hunk 1]
```diff
@@ -36,21 +36,14 @@ M.start = function(args)
 	for k, v in pairs(M.opts) do
 		if k == "command" or k == "gobble" then
 			-- no-op
-		elseif k == "language" then
-			table.insert(cmdline, '--language')
+		elseif k == "language" or k == "output" or k == "window_title" then
+			table.insert(cmdline, "--" .. string.gsub(k, "_", "-"))
 			if type(v) == "function" then
-				table.insert(cmdline, v())
+				value = v()
```
Comment (michaelrommel) [inline lua/silicon.lua:42]: In line 35, you could add
```
local value = nil
```

File: lua/silicon.lua [hunk 2]
```diff
@@ -36,21 +36,14 @@ M.start = function(args)
...
-		elseif k == "output" then
-			table.insert(cmdline, '--output')
-			if type(v) == "function" then
-				filename = v()
-			else
-				filename = v
+				value = v
 			end
-			table.insert(cmdline, filename)
+			table.insert(cmdline, value)
```
Comment (michaelrommel) [inline lua/silicon.lua:46]: Then here:
```
if k == "output" then filename = value end
```

**Review by michaelrommel — CHANGES_REQUESTED** [review]
I'd appreciate if you made the changes, then your contribution can be retained, instead of me doing this...
I also thought about adding this flag to the default_opts table, but opted against it, because then the plugin would only work for people on the master branch...

File: lua/silicon.lua (hunk 1 above)
Comment (andresatierf) [reply lua/silicon.lua:43]: Fixed

File: lua/silicon.lua (hunk 2 above)
Comment (andresatierf) [reply lua/silicon.lua:47]: That's my bad, didn't realize it was being used elsewhere. Fixed

**Review by michaelrommel — APPROVED** [review]

**Pull request merged**